regulations/templates/regulations/chrome.html

```diff
@@ -138,12 +138,15 @@ <h2 class="toc-type">Search</h2>
         <div class="drawer-content">
             <form action="{% url 'chrome_search' reg_part %}" method="GET" role="search">
                 <div class="search-box">
-                    <input type="text" name="q" value="{{ q }}" id="search-input" /><button type="submit"><i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong></button>
+                    <input type="text" name="q" value="{{ q }}" id="search-input" title="Search term"/>
+                    <button type="submit">
+                        <i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong>
```

Comment:
-site was originally written with there was a debate over whether or not `i` should be used to indicate "icon" (see, e.g. http://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span). It _was_ the suggested system to use with CFPB's icon font. Times change, though.

I'd be for making an issue to replace these throughout, but I don't think that should hold back this PR